src/emulator/auth/operations.ts

```diff
@@ -2543,6 +2556,123 @@ function parsePendingCredential(
   return { user, signInProvider };
 }
 
+function createTenant(
+  state: ProjectState,
+  reqBody: Schemas["GoogleCloudIdentitytoolkitAdminV2Tenant"]
+): Schemas["GoogleCloudIdentitytoolkitAdminV2Tenant"] {
+  assert(state instanceof AgentProjectState, "((Can only create tenant in agent project.))");
+
+  const formattedPhoneNumbers: { [key: string]: string } = {};
+  if (reqBody.testPhoneNumbers) {
+    Object.entries(reqBody.testPhoneNumbers).forEach(([phoneNumber, value]) => {
+      formattedPhoneNumbers[formatPhoneNumber(phoneNumber)] = value;
+    });
+  }
+
+  const tenant = {
+    displayName: reqBody.displayName,
+    allowPasswordSignup: reqBody.allowPasswordSignup,
+    enableEmailLinkSignin: reqBody.enableEmailLinkSignin,
+    enableAnonymousUser: reqBody.enableAnonymousUser,
+    disableAuth: reqBody.disableAuth,
+    testPhoneNumbers: formattedPhoneNumbers,
+    mfaConfig: reqBody.mfaConfig,
+    tenantId: "", // Placeholder until one is generated
+  };
+
+  return state.createTenant(tenant);
+}
+
+function formatPhoneNumber(phoneNumber: string) {
+  assert(
+    phoneNumber.length < PHONE_NUMBER_MAX_LENGTH,
```

Comment:
`PHONE_NUMBER_MAX_LENGTH` implies phoneNumber can also equal that length. Should it instead be `<=` ?

Reply:
Ah you're right, I meant for it to be <=, thanks for the catch!